uui/components/layout/VirtualList.tsx

```diff
@@ -30,7 +30,7 @@ type VirtualListRenderRows<List extends HTMLElement = any> = {
 interface BaseVirtualListProps
     extends IHasCX,
     IEditable<VirtualListState>,
-    IHasRawProps<HTMLAttributes<HTMLDivElement>> {
+    IHasRawProps<ScrollbarProps> {
```

Comment:
Since `ScrollbarProps` already has all `HTMLAttributes`, and since `rawProps` passed as is to nested `Scrollbars` component, it's better to set `rawProps` type to `ScrollbarProps` instead, to fix inconsistency between overlapping DOM methods of `HTMLDivElement` and `ScrollbarProps` interfaces